synapse/tools/cellauth.py

```diff
@@ -267,13 +268,21 @@ async def main(argv, outprint=None):
     global outp
     outp = outprint
 
+    fini = None
```

Comment:
could this be cleaner as a ``contextlib.ExitStack`` ?

Reply:
not sure exactly what that would look like.  lets chat about it.